### ISSUE TITLE
Various tile image errors in multiple Scenes with Extras + some placed creature tokens not properly opening its sheet

### ISSUE THREAD

**Issue by chaoticnote** [tianevitt/tianzes-gauntlight-extras#9]
Hello! I've just downloaded this today. I'm liking what I'm seeing so far, but I have noticed at least a couple Scenes with tile image errors.

### I - Yldaris with Extras
![image](https://github.com/user-attachments/assets/cfbf1de4-5518-4011-b8a3-c0333353ddd0)
![image](https://github.com/user-attachments/assets/b30a528c-5f52-48e0-9d94-ab927433a53b)
![image](https://github.com/user-attachments/assets/7cf92b62-243a-4f1f-a0d6-76c1f01d6af5)

### Arcana Obcsura
![image](https://github.com/user-attachments/assets/1801bfdf-e6ec-4f97-811f-f6cb33673fcf)

### Some creature tokens not properly opening up their sheets
Another thing I've discovered is that a few creature tokens of this module's remade Scenes don't properly open up their actor sheets when double-clicked on, such as the two Gugs and the Witchfire Warden in "G17. Stasis Chambers" of _G - Prison with Extras_. The two Gugs placed here are also not the correct token size for a Gug. There's a distinct possibility that other creature tokens may also have errors.
![image](https://github.com/user-attachments/assets/68949949-a255-4522-84e9-53667927c24f)

### Current Foundry Build, PF2E system version, and add-on module versions
![image](https://github.com/user-attachments/assets/6d4a8e39-bad2-4af7-8c2d-75fa00ec7cc8)
![image](https://github.com/user-attachments/assets/059a6b1d-d84c-4b44-9e85-baa3b00c611c)

**Comment by tianevitt** [comment]
Yikes, I guess I need to fix where these images are pointed. This sort of thing is so hard to spot without a separate test machine, which I need to get set up. 

I should have time to work on this during the thanksgiving weekend. Thanks for the detailed report!

**Comment by chaoticnote** [comment]
Is there a way to import the module without losing maps already imported?

What I mean by that is that in my experience, importing a module with scenes that have already been imported before resets everything to how the module is. That means any creature and character tokens that were deleted off the scene are reset back to the way they are, and that's inconvenient when updating in the middle of the adventure.

**Comment by tianevitt** [comment]
None that I know of. The easiest workaround is to move the maps you have changed ( and anything else in the other folders ) to another folder. Then import the adventure, letting the Adventure Document overwrite the "Gauntlight Extras" folder.

Then, delete the new maps you don't need, and move your maps back to the Gauntlight Extras folder.

I wish the the default behavior were to only overwrite what has not changed. For this reason, I have considered just using separate compendiums, but that requires a lot of work for the user. There are also other issues with the adventure doc that were supposed to be fixed (the thumbnail issue) but is still broken for me and requires extra setup.

Somehow, the Abomination Vaults adventure doc can work around this issue. I have not figured it out yet. 

My apologies for the delay in getting this released. Life intruded. The good news is that I fixed what you found, discovered some other issues, and fixed them all. I just need to test them on my new testing machine. Planning to do that tonight and get it released.

**Comment by tianevitt** [comment]
Just an addendum - I ran the adventure update on my test box and it definitely did not overwrite everything, even though it said it would. I had to delete all the folders of the various document types before I got the new content to install.

So the easiest way to keep what you have and bring in what's new is to rename your Gauntlight Extras folders, import the adventure, and then pull your old maps into the newly created Gauntlight Extras folders.

I think for the other folder types, it will update what is there, but I have not specifically confirmed that.
